VISION.md

```diff
@@ -101,15 +101,15 @@ DataCore, Inc is a CNCF Silver member. DataCore support's CNCF extensively and f
 The OpenEBS project produces the OpenEBS product referred to as **```Open Source STANDARD```** (OSS).<BR>
 Many commercial and non-commercial vendors have incorporated, or wish to incorporate a version of OpenEBS as a component or integrated service within their own product offering, including Microsoft Azure, DataCore and Civo. [see ADOPTERS file](./ADOPTERS.md)
 
-The OpenEBS project and CNCF recognize the importance of supporting this, and maintain a conformance program for vendors to use OpenEBS technology, and OpenEBS marks.
+The OpenEBS project and CNCF recognize the importance of supporting this, and maintain a conformance program for vendors to use OpenEBS technology, and OpenEBS brand copyrights.
```

Comment:
The correct legal term for "brand copyright"... is actually the legal word... "Marks". Please revert back to that.

Reply:
Modified accordingly.